File: 240502/청소당번 정하기/cleaning-numbering.js
const fs = require('fs');
const n = Number(fs.readFileSync(0).toString());

let classCnt = 0;
let passageCnt = 0;
let restRoomCnt = 0;

for(let i = 1; i <= n; i++){
    if(i % 12 === 0){
        restRoomCnt++;
    }else if(i % 3 === 0){
        passageCnt++;
    }else if(i % 2 === 0){
        classCnt++;
    }
}

console.log(classCnt + ' ' + passageCnt + ' ' + restRoomCnt);

File: 240502/청소당번 정하기/cleaning-numbering.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        int classCnt = 0;
        int passageCnt = 0;
        int restRoomCnt = 0;

        for(int i = 1; i <= n; i++){
            if(i % 12 == 0){
                restRoomCnt++;
            }else if(i % 3 == 0){
                passageCnt++;
            }else if(i % 2 == 0){
                classCnt++;
            }
        }

        System.out.println(classCnt + " " + passageCnt + " " + restRoomCnt);
    }
}